Content.Server/DeviceLinking/Components/ObjectSensorComponent.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Server.DeviceLinking.Systems;
+using Content.Shared.DeviceLinking;
+using Content.Shared.Tools;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.DeviceLinking.Components;
+
+/// <summary>
+/// An anchored subfloor thing that detects entities that are on the same tile.
+/// </summary>
+[RegisterComponent, Access(typeof(ObjectSensorSystem))]
+public sealed partial class ObjectSensorComponent : Component
+{
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public ProtoId<SinkPortPrototype> OutputPort1 = "ObjectSensor1Object";
```

Comment:
sink is input

Reply:
DAMNIT

Reply:
the real question is why does it work :trollface:

Reply:
because ProtoId is just string with linting

Reply:
i love lint